nf_core/create.py

```diff
@@ -507,59 +504,12 @@ def fix_linting(self):
 
     def make_pipeline_logo(self):
         """Fetch a logo for the new pipeline from the nf-core website"""
-
-        logo_url = f"https://nf-co.re/logo/{self.template_params['short_name']}?theme=light"
-        log.debug(f"Fetching logo from {logo_url}")
-
-        email_logo_path = self.outdir / "assets" / f"{self.template_params['name_noslash']}_logo_light.png"
-        self.download_pipeline_logo(f"{logo_url}?w=600&theme=light", email_logo_path)
+        email_logo_path = Path(self.outdir / "assets")
+        Logo().create(text=self.template_params["name_noslash"], dir=email_logo_path, theme="light")
```

Comment:
Feels slightly odd to be using the main class like this, rather than creating an object from it. eg. this kind of thing:

```python
pipeline_logo = Logo()
pipeline_logo.create(...)
```

Like, if you're just calling it directly like this then why not just have a function instead of a Class?

Reply:
I wasn't sure if we didn't want to add more functions so decided to go for a class. now in the end, it actually is not needed. will make it only a function